checksec

```diff
@@ -911,18 +930,10 @@ kernelcheck() {
     fi
   fi
 
-  echo_message "  Exec Shield:                            " """" ""
```

Comment:
why is this being removed?

Reply:
@slimm609 Exec Shield is obsolete and non-existent in non-ancient kernels, see https://github.com/slimm609/checksec.sh/pull/124